build.rs

```diff
@@ -7,4 +7,16 @@ fn main() {
             err
         );
     }
+
+    use std::env;
+    if let Ok(target) = env::var("TARGET") {
+        match target.as_ref() {
+            "asmjs-unknown-emscripten" => { },
+            "wasm32-unknown-emscripten" => { },
+            "wasm32-unknown-unknown" => { },
+            _ => println!("cargo:rustc-cfg=parallel"),
```

Comment:
I don't understand what this file does, except printing?

Reply:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script